verification/cmd/verification-service/README.md

```diff
@@ -21,6 +21,19 @@ configuration:
 
 The verifier currently doesn't support any configuration.
 
+### Config files
+
+There are two config files in this directory:
+
+- `config.yaml` is designed to be used when running `verification-service`
+  directly form this directory.
+- `config-docker.yaml` this is the file that is designed to be used when running
+  inside the debug docker container. See [debugging docker
+  deployment](/deployments/docker/README.md#Debugging). The `debug` command
+  inside the debug container will automatically use it. If running the
```

Comment:
Line 33 starting `If running the ..` is slightly confusing, as for running the executable directly, we use `config.yaml` ??

Reply:
That means directly, as opposed to via `debug` command, but still inside the docker container.

Reply:
I've added clarifaction to hopefully make it less confusing.